templates/aws/workspaces.html

```diff
@@ -1,96 +1,156 @@
-        {{
-          image(
-            url="https://assets.ubuntu.com/v1/55bb92be-aws_workspaces.png",
-            alt="",
-            height="215",
-            width="603",
-            hi_def=True,
-            loading="auto"
-          ) | safe
-        }}
+      <div class="col p-strip is-shallow">
+        <div class="p-image-container--16-9 is-highlighted">
+          {{ image (url="https://assets.ubuntu.com/v1/b914870e-aws-f7f7f7.png",
+                    alt="AWS Workspaces",
+                    width="300",
+                    height="200",
+                    hi_def=True,
+                    loading="auto") | safe
+          }}
+        </div>
       </div>
     </div>
-  </div>
-</section>
-
-<section class="p-strip">
-  <div class="u-fixed-width">
-    <div class="p-strip is-shallow u-no-padding--top">
-      <h2>What are Ubuntu WorkSpaces?</h2>
+    <div class="row">
+      <hr class="is-muted" />
+      <div class="col">
+        <a href="https://aws.amazon.com/workspaces/pricing/?nc=sn&loc=3"
+           class="p-button--positive">Launch Today</a>
+      </div>
     </div>
-  </div>
-
-  <div class="row p-divider">
-    <div class="col-4 p-divider__block">
-      <h3 class="p-heading--4">Ubuntu Desktop 22.04 LTS</h3>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">A common OS for development, testing and deployment</li>
-        <li class="p-list__item is-ticked">Linux 5.15 kernel alongside the latest toolchains for Python, PGP, Rust and Go</li>
-        <li class="p-list__item is-ticked">Target platform for cutting edge applications in fields like Data Science, AI/ML, Cloud and IoT</li>
-        <li class="p-list__item is-ticked">Active Directory integration</li>
-      </ul>
+  </section>
 
-      <p>Read more about <a href="/desktop/developers">Ubuntu Desktop for developers</a></p>
+  <section class="p-section">
+    <div class="row">
+      <hr />
+      <div class="col">
+        <div class="p-section--shallow">
+          <h2>What are Ubuntu WorkSpaces?</h2>
+        </div>
+      </div>
     </div>
-
-    <div class="col-4 p-divider__block">
-      <h3 class="p-heading--4">Remote onboarding for developers</h3>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">Easily onboard remote workers and contractors and get them up and running quickly</li>
-        <li class="p-list__item is-ticked">Portable desktops for any device accessed via web browser or native client applications</li>
-        <li class="p-list__item is-ticked">Pay as you go without the need to procure high-end hardware</li>
-      </ul>
-
-      <p>Read more about <a href="https://aws.amazon.com/workspaces/">Amazon WorkSpaces</a></p>
+    <div class="row">
+      <div class="col-start-large-4 col-9">
+        <hr class="p-rule is-muted" />
+        <div class="row">          
+          <div class="col-3">
+            <h5>Ubuntu Desktop 22.04 LTS</h5>
```

Comment:
it's better to not use different headings together, so `<h3 class="p-heading--5">` would be better. Please adjust it in the below sections too.